src/big-bang-parser/big_bang_lexer.mll

```diff
@@ -63,6 +63,7 @@ rule read = parse
 
   | "=?"    { OPERATOR_EQUALITY }
   | "not=?" { OPERATOR_INEQUALITY }
+  | "<?"    { OPERATOR_LESS_THAN }
```

Comment:
That's a fine question.  I'm still a little concerned about `=?` given that it takes more physical effort to type than `==`, but I'm sure everyone can adjust.  In the case of `<?`, I'd lean toward just writing `<` but I'm not sure if that's tradition speaking.  Typing `<?` isn't really that hard; my larger complaint there is that it's a bit more difficult to read (e.g. `x < y` vs. `x <? y`).